libmachine/engine/engine.go

```diff
@@ -5,6 +5,7 @@ const (
 )
 
 type Options struct {
+	EngineName       string
```

Comment:
This structure clearly refers to the container runtime used inside the machine..
Since we're now supporting multiple ones, I think it would make sense to maintain a reference to which container runtime has been chosen for the machine here, instead of relying on minikube's config package at this level